server/src/main/java/org/opensearch/identity/AuthenticationManager.java

```diff
@@ -0,0 +1,61 @@
+package org.opensearch.identity;
```

Comment:
Why did you pick `.identity` over `.auth`?

Reply:
identity is a dictionary word and that appeals to me.